packages/amazonq/package.json

```diff
@@ -454,13 +441,13 @@
                 "command": "aws.amazonq.login",
                 "title": "%AWS.command.login%",
                 "category": "%AWS.title%",
-                "enablement": "isCloud9 || !aws.isWebExtHost",
```

Comment:
Can you double check whether Q standalone should be supported in C9 and web mode? If yes, then this PR needs to be tested against C9 and web

Reply:
Q does not need to be supported in C9. Webmode is uncertain, but out of scope for the immediate future. We will revisit these commands.